### PR TITLE
add resnet

### PR DESCRIPTION
**Pull request by jacquesqiao**
project: https://github.com/PaddlePaddle/Paddle/issues/4531

### CODE REVIEW

File: python/paddle/v2/framework/tests/test_image_classification_train.py [hunk 1]
```diff
@@ -123,11 +240,11 @@ def conv_block(input,
...
 
         loss = np.array(outs[0])
-        # print("pass_id:" + str(pass_id) + " batch_id:" + str(batch_id) +
-        #       " loss:" + str(loss))
+        print("pass_id:" + str(pass_id) + " batch_id:" + str(batch_id) +
+              " loss:" + str(loss))
         batch_id = batch_id + 1
 
-        if batch_id > 1:
+        if batch_id > 10:
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_image_classification_train.py]: The following comment tells that two minibatches is sufficient to make the test conclusion. Should we change the comment as well?

File: python/paddle/v2/framework/tests/test_image_classification_train.py [hunk 2]
```diff
@@ -75,8 +189,11 @@ def conv_block(input,
     data_type='int64',
     program=program,
     init_program=init_program)
-vgg_net = vgg16_bn_drop(images, program, init_program)
-predict = layers.fc(input=vgg_net,
+# net = vgg16_bn_drop(images, program, init_program)
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_image_classification_train.py]: Do we need a comment to hint the users switch between VGG and ResNet?

File: python/paddle/v2/framework/tests/test_image_classification_train.py (hunk 1 above)
Comment (jacquesqiao) [reply python/paddle/v2/framework/tests/test_image_classification_train.py]: reverted to 2, 10 is for self-test.

File: python/paddle/v2/framework/tests/test_image_classification_train.py (hunk 2 above)
Comment (jacquesqiao) [reply python/paddle/v2/framework/tests/test_image_classification_train.py]: done

**Pull request merged**